src/api/app/models/user.rb

```diff
@@ -66,7 +66,18 @@ def create_home_project
     return if [ "_nobody_", "Admin" ].include? self.login
     # may be disabled via Configuration setting
     return unless can_create_project?(self.home_project_name)
-    Project.find_or_create_by(name: self.home_project_name)
+    project = Project.find_by(name: self.home_project_name)
+    unless project
+      project = Project.create(name: self.home_project_name)
+      # TODO: This is neccesarry for registering. I find this ugly...
+      previous_user = User.current
+      User.current = self
```

Comment:
Other Users can create Users (Admins). No User.current can create Users (Sign Up). Both trigger creation of the Project on the backend now which needs User.current to be set...